src/main/resources/avro/metadata.avdl

```diff
@@ -9,6 +9,32 @@ protocol Metadata {
 import idl "common.avdl";
 
 /**
+  An ontology term describing an attribute. (e.g. the phenotype attribute
+  'polydactyly' from HPO)
+  */
+  record OntologyTerm {
```

Comment:
@macieksmuga , @david4096 - I see this PR uses a different version of OntologyTerm to the Variant Annotation PR.

Here we have:
{"ontologySourceName":"Sequence Ontology",
"ontologySourceID":"http://www.sequenceontology.org/browser/release_2.5.3/term/SO:0001583",
"ontologySourceVersion":"release_2.5.3"}

and in #519 we have:
{"ontologySource":"Sequence Ontology",
"name":"missense_variant",
"id":"http://www.sequenceontology.org/browser/release_2.5.3/term/SO:0001583"}

The former is from the current metadata branch and the latter was in the  master branch last July.

@mbaudis - was it intentional to drop the name from the record? It seems a useful piece of information for the API to return.